packages/orchestration/src/proposals/start-stakeOsmo.js

```diff
@@ -47,11 +47,10 @@ export const startStakeOsmo = async ({
 
   const chainHub = makeChainHub(await agoricNames);
 
-  const agoric = await chainHub.getChainInfo('agoric');
-  const osmosis = await chainHub.getChainInfo('osmosis');
-  const connectionInfo = await chainHub.getConnectionInfo(
-    agoric.chainId,
-    osmosis.chainId,
+  const [_, osmosis, connectionInfo] = await getChainsAndConnection(
+    chainHub,
+    'agoric',
+    'osmosis',
```

Comment:
Great addition 👏 .

Noting that this doesn't help limit roundtrips for scenarios where we only have a `chainId`, like here:
https://github.com/Agoric/agoric-sdk/blob/8f1c7bdcaba96f5218ac4b4639c246773aac0aa9/packages/orchestration/src/exos/local-chain-account-kit.js#L241-L245

This is motivating factor for considering including `chainName` on [ChainAddress](https://github.com/Agoric/agoric-sdk/blob/8f1c7bdcaba96f5218ac4b4639c246773aac0aa9/packages/orchestration/src/orchestration-api.ts#L58-L64)

Reply:
Oh, nice! https://github.com/Agoric/agoric-sdk/pull/9498